sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDB.scala

```diff
@@ -144,26 +156,28 @@ class RocksDB(
    * Put the given value for the given key and return the last written value.
```

Comment:
Please also change the comment correspondingly.

Reply:
Nice finding! Missed that.